tests/integration/telemetry/stats/prometheus/http/customizemetrics/testdata/attributegen_envoy_filter.yaml

```diff
@@ -0,0 +1,53 @@
+apiVersion: networking.istio.io/v1alpha3
+kind: EnvoyFilter
+metadata:
+  name: istio-attributegen-filter
+spec:
+  workloadSelector:
+    labels:
+      app: server
+  configPatches:
+    - applyTo: HTTP_FILTER
+      match:
+        context: SIDECAR_INBOUND
+        listener:
+          filterChain:
+            filter:
+              name: "envoy.http_connection_manager"
+              subFilter:
+                name: "istio.stats"
+      patch:
+        operation: INSERT_BEFORE
+        value:
+          name: istio.attributegen
+          typed_config:
+            "@type": type.googleapis.com/udpa.type.v1.TypedStruct
+            type_url: type.googleapis.com/envoy.extensions.filters.http.wasm.v3.Wasm
+            value:
+              config:
+                configuration:
+                  "@type": "type.googleapis.com/google.protobuf.StringValue"
+                  value: |
+                    {
+                      "attributes": [
```

Comment:
Thanks Xinnan.
Can you also add a negative test case that misses say the request_operation?

In the operation you can add request.path as an additional condition. And then send 2 requests, one with and one without the correct path.

Reply:
added two call, one with correct method one without